TF_Application/Assets/TouchFree_Application/Scripts/TransparentWindow.cs

```diff
@@ -166,18 +166,26 @@ void Update()
 #if !UNITY_EDITOR
 		    if (clickThroughEnabled)
```

Comment:
The indentation is a bit off for this if statement (including it's closing brace)

Reply:
I think it's because there's a mixture of tabs and spaces on those lines, so I guess github renders that differently to your IDE.